exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -329,14 +334,27 @@ class Join(
  */
 @Suppress("TooManyFunctions")
 open class Table(name: String = "") : ColumnSet(), DdlAware {
+
+    val schema: String
+
+    init {
+        schema = if (name.contains(".")) {
+            name.substringBefore(".")
+        } else {
+            ""
+        }
```

Comment:
this is leaves room for possible unexpected behavior, when the table name contains multiple dots

Reply:
My suggestions would be, this would treat the table name always the same:
        schema = if (name.contains(".")) {
            name.substringBeforeLast(".")
        } else {
            ""
        }

Reply:
Thanks for the suggested changes @BierDav !